autoendpoint/src/server/extractors/new_channel_data.rs

```diff
@@ -0,0 +1,10 @@
+use uuid::Uuid;
+
+/// The data provided when creating a new channel for an existing user. Extract
+/// from the request via the `Json` extractor.
+#[derive(serde::Deserialize, Default)]
+pub struct NewChannelData {
+    #[serde(rename = "channelID")]
+    pub channel_id: Option<Uuid>,
```

Comment:
See note later about why we may want to store this as a string instead of a UUID.